src/Npgsql/TypeHandlers/CompositeHandlers/CompositeParameterHandler.Generic.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Reflection;
+using System.Threading.Tasks;
+using Npgsql.TypeHandling;
+
+namespace Npgsql.TypeHandlers.CompositeHandlers
+{
+    sealed class CompositeParameterHandler<T> : CompositeParameterHandler
```

Comment:
The standard file naming for this would be ``CompositeParameterHandler`.cs`` (like ``NpgsqlParameter`.cs`` which is the generic version of `NpgsqlParameter.cs`)

Reply:
Okay, will do. But what if I have many generics? How should files be named?

Reply:
IIRC the standard is X`1.cs, X`2.cs and so on.